.mvn/maven.config

```diff
@@ -1 +1 @@
--DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -s settings.xml -P spring
+-DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -P spring
```

Comment:
You can add that on the command line if you ever need it, but you would only need it for e.g. travis where you want to set credentials and don't have a user. You don't need it for any routine stuff like building and installing locally (IMO the wrapper configuration needs to be aligned with users who clone the code from github and just want to build locally).